checks/check_extra93

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Prowler - the handy cloud security tool (copyright 2018) by Toni de la Fuente
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+CHECK_ID_extra93="9.3"
+CHECK_TITLE_extra93="[extra93] Check if aws cloudwatch has allowed sharing with other accounts (Not Scored) (Not part of CIS benchmark) (Custom Check)"
```

Comment:
This check is checking that the role to share cloudwatch is created or not.
If it's created, shouldn't it mean that it's been created on purpose?
The title says that it's checking if cloudwatch event are allowed to be shared with other accounts, shouldn't it check that are closed instead? What if I'm aggregating cloudwatch stuff in another account? I will be failing this check

Reply:
Yes the role can be created on purpose. The intention was to list cloud-watch in prowler if it has allowed cross account access, so rather than failing the check we can make it as Info.

Please explain a bit more "Shouldn't it check that are closed instead?" do you mean the events ?

Reply:
I think @toniblyx can add more to this :)